tailcfg/tailcfg.go

```diff
@@ -528,7 +528,9 @@ type Hostinfo struct {
 	ShareeNode      bool           `json:",omitempty"` // indicates this node exists in netmap because it's owned by a shared-to user
 	NoLogsNoSupport bool           `json:",omitempty"` // indicates that the user has opted out of sending logs and support
 	WireIngress     bool           `json:",omitempty"` // indicates that the node wants the option to receive ingress connections
+	Machine         string         `json:",omitempty"` // the host's machine type (uname -m)
 	GoArch          string         `json:",omitempty"` // the host's GOARCH value (of the running binary)
+	GoArchVar       string         `json:",omitempty"` // GOARM, GOAMD64, etc
```

Comment:
The previous comment says it is the running binary, I think this one is the compiled binary?

Reply:
clarified all